sdk/file_sdk/Common/Common.pro

```diff
@@ -2,23 +2,27 @@ REPO_ROOT = $$PWD/../../..
 DESTDIR   = $$REPO_ROOT/bin/filesdk
 TARGET    = modfilecommon
 
-
 TEMPLATE  = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
-INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI
+INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI $$REPO_ROOT/sdk/rms_sdk/ModernAPI $$REPO_ROOT/sdk/file_sdk
```

Comment:
we should not use .. in #include

---
In reply to: [134661416](https://github.com/AzureAD/rms-sdk-for-cpp/pull/134#discussion_r134661416) [](ancestors = 134661416)